src/chain/transaction.js

```diff
@@ -208,6 +211,25 @@ export default class Transaction {
     return null
   }
 
+  getSender1() {
+    return this._getSender(0)
+  }
+
+  getSender2() {
+    return this._getSender(1)
+  }
+
+  _getSender(i) {
+    if (this._inputNull(i)) {
+      return null
+    }
+
+    const vrs = utils.fromRpcSig(this.raw[11 + i]) // parse {v,r,s} from sig
```

Comment:
These magic numbers make code hard to follow. Can we define properties to access each part perhaps?

Reply:
Will fix them.